### PR TITLE
feat: devnet start 

### PR DESCRIPTION
**Pull request by tippenein**
#1101

- [x] moves `integrate` to `devnet start` but keep integrate in place for now
- [x] adds the --package flag to specify the direct contents of `devnet package`
- [x] documentation?
- [x] add warn message to `integrate` usage

### CODE REVIEW

File: components/clarinet-cli/src/frontend/cli.rs [hunk 1]
```diff
@@ -160,6 +160,10 @@ enum Devnet {
     /// Generate package of all required devnet artifacts
     #[clap(name = "package", bin_name = "package")]
     Package(DevnetPackage),
+
+    /// Start a local Devnet network for interacting with your contracts from your browser
+    #[clap(name = "start", bin_name = "start")]
+    DevnetStart(DevnetStart),
```
Comment (hugocaillard) [inline components/clarinet-cli/src/frontend/cli.rs:166]: To be moved around L98
Just to keep both version close to each other

File: components/clarinet-cli/src/frontend/cli.rs [hunk 2]
```diff
@@ -422,6 +426,9 @@ struct Integrate {
         conflicts_with = "use-on-disk-deployment-plan"
     )]
     pub use_computed_deployment_plan: bool,
+    /// Path to Package.json
+    #[clap(long = "package")]
+    pub package: Option<String>,
```
Comment (hugocaillard) [inline components/clarinet-cli/src/frontend/cli.rs:431]: I think I'm missing some context here.
What is the package.json used for?

**Review by hugocaillard — COMMENTED** [review]
Thanks @tippenein 
A couple of questions/remarks

File: components/clarinet-cli/src/frontend/cli.rs (hunk 2 above)
Comment (tippenein) [reply components/clarinet-cli/src/frontend/cli.rs:431]: last bullet from #1101 

> This new --package flag will allow a user to start a devnet using only a clarinet project package (created from clarinet devnet package). The user can then start a devnet without having to run clarinet integrate from a specific directory - they just need the package file, and it has everything they need.

Comment (hugocaillard) [reply components/clarinet-cli/src/frontend/cli.rs:431]: ah yes ofc. My bad, I was thinking about a node.js `package.json` and got confused. Sounds good then, thx

**Comment by hugocaillard** [comment]
Hey @tippenein 
Since you still have the "testing" task unchecked in the description, I was wondering if this is ready for review or if you'll still work on this PR?

**Comment by tippenein** [comment]
It's up to you @hugocaillard. I don't have a lot of context on how to test this properly, but I could give it a shot if CodeCov needs to be appeased

**Comment by hugocaillard** [comment]
@tippenein I've been working of fixing the CI and coverage report this week. Most of the coverage reports were kind of incorrect so we don't care too much about that for now. But seen we will.
If you think the PR's ready, I'll review it today or early next week 👍

File: components/clarinet-cli/src/frontend/cli.rs [hunk 3]
```diff
@@ -422,6 +426,9 @@ struct Integrate {
         conflicts_with = "use-on-disk-deployment-plan"
     )]
     pub use_computed_deployment_plan: bool,
+    /// Path to Package.json
+    #[clap(long = "package")]
```
Comment (hugocaillard) [inline components/clarinet-cli/src/frontend/cli.rs:430]: It should probably conflict with the two others properties right (it shouldn't be possible to specify a package as well as manifest_path or use_computed_deployment_plan)?

**Review by hugocaillard — CHANGES_REQUESTED** [review]
@tippenein 
Almost lgtm, just a nitpick
I'll try to get this merged soon. i also have an upocuming PR that upgrade clap to v4.x so I'd like to merge this on first

File: components/clarinet-cli/src/frontend/cli.rs (hunk 3 above)
Comment (tippenein) [reply components/clarinet-cli/src/frontend/cli.rs:430]: True! nice catch

Comment (tippenein) [reply components/clarinet-cli/src/frontend/cli.rs:430]: I wonder too about the description. Could be better.. Any idea?

**Review by hugocaillard — APPROVED** [review]
LGTM!
Thanks @tippenein

**Pull request merged**